tidb-cloud/integrate-tidbcloud-with-netlify.md

```diff
@@ -32,32 +32,184 @@ For Dedicated Tier clusters, make sure that the traffic filter of the cluster al
 
-## Connect via manually setting environment variables
+## Get example projects and set connection string
 
-1. Follow the steps in [Connect to a TiDB Cloud cluster via standard connection](/tidb-cloud/connect-via-standard-connection.md) to set a password and get the connection information of your TiDB cluster.
+We provide a fullstack app in TypeScript with Next.js using React and Prisma Client here. It's a simple blog site where you can post and delete your own blog. All the content is stored in TiDB cloud through prisma.
 
-    > **Note:**
-    >
-    > For Dedicated Tier clusters, make sure that you have also set the **Allow Access from Anywhere** traffic filter in this step.
+### Fork example and clone it to your own space
 
-2. Go to your **Netlify dashboard** > **Netlify project** > **Site settings** > **Environment Variables**, and then [update variables](https://docs.netlify.com/environment-variables/get-started/#update-variables-with-the-netlify-ui) according to the connection information of your TiDB cluster.
+Navigate to [Fullstack Example with Next.js and Prisma](https://github.com/tidbcloud/nextjs-prisma-example), fork it to your own GitHub repository.
 
-    Here we use a Prisma application as an example. The following is a datasource setting in the Prisma schema file for a TiDB Cloud Serverless Tier cluster:
+Then, use the following command to clone it.
 
+```shell
+git clone https://github.com/${your_username}/nextjs-prisma-example.git
+cd nextjs-prisma-example/
+```
+
+### Get TiDB Cloud database connection string
+
+<SimpleTab>
+<div label="TiDB Cloud CLI">
+
+> **Note:**
+>
+> Only **Serverless Tier** can get Prisma connection string from TiDB Cloud CLI. If your database is Dedicated Tier, please use TiDB Cloud console to get.
+>
+
+```shell
+ticloud cluster connect-info
+```
+
+Then, follow the prompts to select your cluster and system and select Prisma as client.
+
+```
+Choose the cluster
+> [x] Cluster0(13796194496)
+Choose the client
+> [x] Prisma
+Choose the operating system
+> [x] macOS/Alpine (Detected)
+
+datasource db {
+  provider = "mysql"
+  url      = "mysql://<User>:<Password>@<Endpoint>:<Port>/<Database>?sslaccept=strict"
+}
+```
+
+> **Note:**
+> 
+> Don't forget to replace the parameters in the connection string.
+> 
+> This example must create a new database, so replace `<Database>` with a name that doesn't exist.
+
+The output is the connection string for Prisma connecting to TiDB Cloud.
+
+For more information about TiDB Cloud CLI, see [TiDB Cloud CLI Reference](./cli-reference.md).
+
+</div>
+<div label="TiDB Cloud console">
+
+Navigate to [TiDB Cloud console](https://tidbcloud.com/). Get the following connection parameters `${host}`, `${port}`, and `${user}` from the connection string in the [**Connect**](/tidb-cloud/connect-via-standard-connection.md) dialog and fill them into the following connection string.
```

Comment:
no password? And in the connection string, better use host instead of endpoint.

Reply:
@zhangyangyu Thanks for your review. I have added `password` and used `host` instead of `endpoint`.